clang/lib/Headers/avx2intrin.h

```diff
@@ -15,8 +15,12 @@
 #define __AVX2INTRIN_H
 
 /* Define the default attributes for the functions in this file. */
-#define __DEFAULT_FN_ATTRS256 __attribute__((__always_inline__, __nodebug__, __target__("avx2"), __min_vector_width__(256)))
-#define __DEFAULT_FN_ATTRS128 __attribute__((__always_inline__, __nodebug__, __target__("avx2"), __min_vector_width__(128)))
+#define __DEFAULT_FN_ATTRS256                                                  \
+  __attribute__((__always_inline__, __nodebug__,                               \
+                 __target__("avx2,no-evex512"), __min_vector_width__(256)))
```

Comment:
Why does the function targeted at avx2 need `no-evex512`?

Reply:
We have defined parts AVX512 intrinsics with `no-evex512` and some of them will call into these AVX2 intrinsics.
Then we are facing a problem that we cannot call them in some cases because we didn't specify `no-evex512` for them.